azure-defender-easm/add-easm-assets-to-tracker/service.py

```diff
@@ -132,53 +133,55 @@ def create_claim(org_id, domain_id, domain_name, txn_col):
             "_from": org_id,
             "_to": domain_id,
             "tags": [{"en": "NEW", "fr": "NOUVEAU"}],
-            "outsideComment": "",
-            "firstSeen": date.today().isoformat(),
+            "firstSeen": datetime.today().astimezone().isoformat(),
         }
 
         try:
-            txn_col.insert(insert_claim)
+            created_claim = txn_col.insert(insert_claim)
             logger.info(f"Successfully created claim for domain: {domain_name}")
+            return created_claim
         except Exception as e:
-            logger.error("Error occured when creating claim for ", e)
+            logger.error(f"Error occured when creating claim for {domain_name}", e)
+            return None
 
-    def log_activity(domain, org_id, trx_col):
+    def log_activity(domain, org_key, txn_col):
         insert_activity = {
-            "timestamp": date.today().isoformat(),
+            "timestamp": datetime.today().astimezone().isoformat(),
```

Comment:
Looks like we actually use the other format for timestamps in the api, this should align with that:

```
datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
```